apps/admin-gui/src/app/admin/pages/admin-user-detail-page/admin-user-detail-page.component.html

```diff
@@ -22,9 +22,15 @@
             data-cy="edit-user-button">
             <mat-icon>edit</mat-icon>
           </button>
-          <button *ngIf="!anonymized" (click)="anonymizeUser()" mat-icon-button>
+          <button
+            *ngIf="!anonymized && !userDeletionForced"
+            (click)="anonymizeUser()"
+            mat-icon-button>
             <mat-icon>no_accounts</mat-icon>
           </button>
+          <button *ngIf="!anonymized && userDeletionForced" (click)="deleteUser()" mat-icon-button>
```

Comment:
It should be possible to delete anonymized users. In case the instance switches to forced deletion after some users were already anonymized.
```suggestion
          <button *ngIf="userDeletionForced" (click)="deleteUser()" mat-icon-button>
```